login.go

```diff
@@ -19,24 +20,74 @@ var loginTopic = &Topic{
 var loginCmd = &Command{
 	Topic:       "login",
 	Description: "Login with your Heroku credentials.",
-	Run: func(ctx *Context) {
-		login()
+	Flags: []Flag{
+		{Name: "sso"},
+		{Name: "interactive"},
+		{Name: "org", HasValue: true},
 	},
+	Run: login,
 }
 
 var authLoginCmd = &Command{
 	Topic:       "auth",
 	Command:     "login",
 	Description: "Login with your Heroku credentials.",
-	Run: func(ctx *Context) {
-		login()
+	Flags: []Flag{
+		{Name: "sso"},
+		{Name: "interactive"},
+		{Name: "org", HasValue: true},
 	},
+	Run: login,
+}
+
+func login(ctx *Context) {
+	if ctx.Flags["sso"] == true {
+		org, _ := ctx.Flags["org"].(string)
+		ssoLogin(org)
+	} else {
+		interactiveLogin()
+	}
+}
+
+func ssoLogin(org string) {
+	for org == "" {
+		org = getString("Enter your organization name: ")
```

Comment:
I guess this comment should really be about line 38 when the org value is initialized.